sdk/metricsadvisor/azure-ai-metricsadvisor/src/samples/java/com/azure/ai/metricsadvisor/administration/MetricsAnomalyAlertConfigOperationsAsyncSample.java

```diff
@@ -86,9 +88,11 @@ public static void main(String[] args) {
         // Update the anomaly alert config.
         Mono<AnomalyAlertConfiguration> updatedAlertConfigMono = fetchAnomalyAlertConfig
             .flatMap(anomalyAlertConfig -> {
-                return advisorAdministrationAsyncClient.updateAnomalyAlertConfig(
+                List<String> hookIds = new ArrayList<>(anomalyAlertConfig.getIdOfHooksToAlert());
+                hookIds.remove(hookId2);
+                return advisorAdministrationAsyncClient.updateAlertConfig(
                     anomalyAlertConfig
-                        .removeHookToAlert(hookId2)
+                        .setIdOfHooksToAlert(hookIds)
```

Comment:
nit: rename suggestion setHookIdsToAlert ?

Reply:
thanks, yes this sounds better, will update.